src/pages/index.js

```diff
@@ -34,50 +36,126 @@ const enableValidation = (formSettings) => {
 
 enableValidation(formSettings);
 
+/* ************************* */
+/* Declaration of Api object */
+/* ************************* */
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  authorization: "a0548a22-3cdf-48eb-9822-7e7daf1b0604",
+});
+
 /* ****************************** */
 /* Declaration of UserInfo object */
 /* ****************************** */
 const userInfo = new UserInfo({
   nameSelector: ".profile__name",
   jobSelector: ".profile__job",
+  avatarSelector: ".profile__picture",
 });
 
 /* ********************** */
 /* Declarations of popups */
 /* ********************** */
+const avatarForm = new PopupWithForm("form[name='avatar-form']", (data) => {
+  avatarForm.submitButton.textContent = "Saving...";
+  api
+    .editProfilePicture(data.link)
+    .then((res) => (res.ok ? true : res.status))
```

Comment:
* According to the task you need to check the response if it's ok and **throw** an error if it's not ok. And this `ok` checking should be only in `Api` to clean up the code of other files. It's the responsibility of `Api`

**Please, correct it everywhere**